inspirehep/modules/orcid/tasks.py

```diff
@@ -126,16 +123,15 @@ def send_to_orcid(sender, api=None):
                     else:
                         api.update_record(author_orcid, token,
                                           'work', orcid_json, str(put_code))
-                    logger.info("Succersfully sent " +
-                                sender.get('control_number') + " to orcid.")
+                    logger.info("Succesfully sent to orcid: ",
+                                sender.get('control_number'))
                 except RequestException as e:
                     print(e.response.text, sender['control_number'])
-                    logger.info("Failed to push " +
-                                sender.get('control_number') + " to orcid.")
+                    logger.error("Failed to push to orcid: ",
+                                 sender.get('control_number'))
                     continue
         except (KeyError, AttributeError, TypeError) as e:
-            logger.info("Failed to convert " +
-                        sender.get('control_number') + " to orcid.")
+            logger.error("Failed to convert: ", sender.get('control_number'))
```

Comment:
If you didn't like `error` there's also `warning`: https://docs.python.org/2/library/logging.html#logging-levels